### DISCUSSION TITLE
Wazuh centos agent logs don't show up on so manager

### DISCUSSION THREAD

**Discussion by mahmoudelhaj** [Security-Onion-Solutions/securityonion#9774] [General]
Hi,,
1) Allow wazuh agent on so-allow (w - 1514)
agent ip: 10.20.1.194

2) Allow registration service  on so-allow (r - 1515)
agent ip: 10.20.1.194

3) (A)dd an agent (A) on Wazuh v3.13.1 Agent manager
name: glpi2
ip: 10.20.1.194

4) (E)xtract key for an agent (E).

5) download wazuh agent from so manager & install it on agent
6) add ip manager & key

![image](https://user-images.githubusercontent.com/29997801/218449121-a5c13a69-bc4c-4caf-af3b-8fcda0300a84.png)
![image](https://user-images.githubusercontent.com/29997801/218449186-edd7d3ff-9603-484e-8cf3-9e5caffc142c.png)
![image](https://user-images.githubusercontent.com/29997801/218449414-b41f13be-6383-4175-8c22-7faac1fa43c4.png)

but no logs show up on so manager!!
![image](https://user-images.githubusercontent.com/29997801/218449549-56aaeac0-e3a7-4a91-9138-6762493fa8ba.png)

**Comment by cm-ops** [comment]
Are there any clues in the agent's `ossec.log`?
